src/components/List/ApplicantCell.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```

Comment:
Just a personal preference, but since these individual cell components are so small I feel it would be easier to have them in one file. Like a library of components in list like in `mapColumnsByRole.`
If there's any complex logic required for any of them, that could be extracted to another file (or at the bottom of this one). What do you think?

Reply:
I disagree. Sorry, I really think it's easier to see things when they are separated in its own file, no matter how big they are. It's not like the bundle size increases, and if we keep thing inside folder to me makes it compact when you are not working with that area of components.